lib/scrabble.rb

```diff
@@ -1,9 +1,21 @@
+require 'pry'
 class Scrabble
+  attr_accessor :points_array,
+                :points
+
+  def initialize
+    @points_array = []
+    @points = 0
+  end
 
   def score(word)
-    1
+    word = prepare(word)
+    return_score(word)
+    combine_score(points_array)
```

Comment:
If `points_array` is an instance variable we don't need to pass it to this method. Can access it directly in the method with `@points_array`.